server/pkg/asset/asset_test.go

```diff
@@ -25,7 +25,7 @@ func TestAsset_AssetType(t *testing.T) {
 		size:        size,
 		previewType: PreviewTypeFromRef(lo.ToPtr(PreviewTypeIMAGE.String())),
 		file:        &File{},
-		hash:        "yyy",
+		uuid:        "xx/xxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/hoge",
```

Comment:
```suggestion
		uuid:        "xxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
```

If you want to get a string as a URL, create it dynamically each time based on the uuid and name; storing it as is in URL format in the DB makes the URL resistant to change.